recaf-ui/src/main/resources/style/recaf.css

```diff
@@ -432,7 +432,6 @@ Text {
   -fx-graphic-text-gap: 6px;
   -fx-text-fill: -color-button-fg;
   -fx-alignment: CENTER;
-  -fx-effect: dropshadow(gaussian, -color-button-shadow, 3px, -2, 0, 1);
```

Comment:
Removed because it would cause the buttons used in Bento's header view to infinitely expand